primer/brand
# Prevents column width from overflowing the River component

## Summary

Closes https://github.com/github/primer/issues/2161

## List of notable changes:

Just a simple style update to sizes River grid columns with `fr` units instead of `%`

## What should reviewers focus on?

How cool and easy CSS grid is.

## Steps to test:

Inspect the River component and confirm that neither column overflows the grid parent.

## Supporting resources (related issues, external links, etc):

n/a

## Contributor checklist:


## Reviewer checklist:


## Screenshots:

<table>
<tr>
<th> Before</th> <th> After </th>
</tr>
<tr>
<td valign="top">

![Screenshot 2023-05-03 at 2 38 40 PM](https://user-images.githubusercontent.com/2313998/236012598-b51a069a-3c8e-42e7-9645-1f353679bb55.png)

 </td>
<td valign="top">

![Screenshot 2023-05-03 at 2 39 14 PM](https://user-images.githubusercontent.com/2313998/236012641-be396d71-f6fa-428c-ab4f-796ddf30ddab.png)


</td>
</tr>
</table>
